nf-test.config

```diff
@@ -0,0 +1,8 @@
+config {
+
+    testsDir "tests"
+    workDir ".nf-test"
+    configFile "nextflow.config"
+    profile "test,docker"
```

Comment:
```suggestion
    profile ""
```

Reply:
we thought not to hardcode profile here, rather have it in the `ci.yml`

Reply:
Could the profile also be part of different test cases? E.g. in this workflow, I would like to test different scenarios:
 * run only downstream with precomputed data
 * run preprocessing with dataset 1
 * run preprocessing with dataset 2

I would have considered to create different profiles for each case. How would you recommend to implement this?

Reply:
profile can indeed be specified for a test suite but the limitation right now is that profile can be specified anywhere only once. as a workaround for pipelines with multiple test profiles like [ampliseq](https://github.com/nf-core/ampliseq/pull/580/files#diff-77dd105fc4d4faef31b616c2d04d5357d123484bb1c92aad7bbac266608ceb47) I just moved all the test config contents to the `when` block so we can then just set `--profile docker` in the ci

Reply:
Ok, so 

```
[...]
when {
    profile = "test_dataset1"
}
[...]
``` 

is not possible?

Reply:
No, it isn't. 

An alternative could be to store the params for the test profile in `yaml/json` files as described [here](https://code.askimed.com/nf-test/testcases/global_variables/). 

Like this the same set of parameters could be used in the `test` profile (which I imaging every pipeline should have regardless of nf-test, because this is a way of quickly testing the pipeline in a new compute environment) and a nf-test test case.

Reply:
For now I'll go with: 
 * keep the `test` profile
 * specify the `test` profile in the CI
 * other test cases can override the parameters set in the `test` profile, but keep default values such as `max_memory` or `max_time`.

Reply:
sounds good